Bottom: a62ae759225360992732b5bdeae08bf3a1aa8b46
Top:    2ef10257086e9abb92eae056cc170b244ccbd7cf
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:53:26 +0000

Disable debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 78e5478..27e7c54 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1253,7 +1253,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                                 copper_appearance,
                                 NULL));
 
-      break;
+      //break;
     }
 
 
@@ -1265,10 +1265,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 //  gui->invalidate_all ();
 
   /* ASSUME THERE IS A POLYGON WHERE WE KNOW WE PUT ONE... */
-  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
-  gui->invalidate_all ();
+//  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
+//  gui->invalidate_all ();
 
-//  poly_Free (&info.poly);
+  poly_Free (&info.poly);
 
   return objects;
 }
